clients/venice-client/src/test/java/com/linkedin/venice/fastclient/meta/RequestBasedMetadataTest.java

```diff
@@ -370,4 +375,31 @@ public void testRequestBasedMetadataStartFailFast() throws IOException {
       Assert.assertThrows(ConfigurationException.class, requestBasedMetadata::start);
     }
   }
+
+  @Test(timeOut = TEST_TIMEOUT)
+  public void testRequestBasedMetadataStartRefreshDuringStoreMigration() throws IOException, InterruptedException {
```

Comment:
I'll rename this. This unit test is to validate whether FC would do refresh when there's exception caught